governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024
```

Comment:
I'll try to stick to titling the items in similar ways to how we've done it before for consistency

```suggestion
title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
```